backend/src/lib/custom-generator.js

```diff
@@ -53,5 +53,46 @@ expressions.filters.convertDateFR = function(input, s) {
     }
 }
 
+// Convert input CVSS criteria into Dutch: {input | criteriaNL}
+expressions.filters.criteriaNL = function(input) {
+    var pre = '<w:p><w:r><w:t>';
+    var post = '</w:t></w:r></w:p>';
+    var result = "Niet gedefinieerd"
+
+    if (input === "Network") result = "Netwerk"
+    else if (input === "Local") result = "Lokaal"
+    else if (input === "Physical") result = "Fysiek"
+    else if (input === "None") result = "Geen"
+    else if (input === "Low") result = "Laag"
+    else if (input === "Medium") result = "Midden"
+    else if (input === "High") result = "Hoog"
+    else if (input === "Critical") result = "Kritiek"
+    else if (input === "Required") result = "Verplicht"
+    else if (input === "Unchanged") result = "Onverandert"
+    else if (input === "Changed") result = "Verandert"
+
+    // return pre + result + post;
+    return result;
+}
+
```

Comment:
Translation is now handled in `backend/src/translate` directory. It would be great if you could use the `fr` directory as a model.